setup.py

```diff
@@ -11,6 +11,7 @@
     "click<8.0",
     "semver~=2.10",
     "python-slugify>=4.0.1",
+    "gcsfs>=0.6.2, <0.7.0",
```

Comment:
If the trick I mentioned below with fsspec works, this could be possibly moved to extra requires 

```
"gcp": {
fsspec... 
}
```

so when using plugin with google cloud you'd install kedro-airflow-k8s[gcp] and ignore that dependency otherwise

Reply:
Yes, on it.